scripts/local-check-test.sh

```
@@ -0,0 +1,328 @@
+        print_error "UI tests failed"
+        return 1
+    fi
+}
+
+# Function to run integration tests
+run_integration_tests() {
+    print_status "Running integration tests with local database..."
+    
+    # Read DATABASE_URL from .env file
+    if [ -f ".env" ]; then
+        export $(grep -E '^DATABASE_URL=' .env | xargs)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)